src/MICore/LaunchOptions.xsd

```diff
@@ -399,6 +389,23 @@
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="ProcessId" type="xs:int" use="optional">
+      <xs:annotation>
+        <xs:documentation>If supplied, the debugger will attach to the process rather than launching a new one. Note that some operating systems will require admin rights to do this.</xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+    <xs:attribute name="CoreDumpPath" type="xs:string" use="optional">
+      <xs:annotation>
+        <xs:documentation>Path to a core dump file for the specified executable.</xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+    <xs:attribute name="DebugChildProcesses" type="xs:boolean" use="optional">
+      <xs:annotation>
+        <xs:documentation>
+          If true, automatically attach debugger to any child processes created.
+        </xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
```

Comment:
This is going to wind up applying to Android and iOS also. Should this really be just pipe/tcp/local?